proseco/guide.py

```diff
@@ -364,24 +364,27 @@ def check_mag_spoilers(cand_stars, ok, stars, n_sigma):
         return np.zeros(len(ok)).astype(bool)
 
     mag_spoiled = np.zeros(len(ok)).astype(bool)
-    # Search the sky around cand stars to see if they are spoiled by other stars
-    coords = SkyCoord(stars['ra'], stars['dec'], unit='deg')
```

Comment:
In my benching, looking for star spoilers via SkyCoord seemed pretty slow and makes the code harder to read.  I think the loop that now replaces it will be a little slower in very dense fields.